xprof/xprof.rb.in

```diff
@@ -754,9 +796,11 @@ def trace_and_on_node_processing(usr_argv)
     end
     # we can kill the session daemon
     lm_lttng_kill_sessiond
+    sampling_daemon&.finalize
```

Comment:
what does this `&` doing? Never saw it before

Reply:
It is safe navigation operator.  Used it to avoid error in case the object is nil.